modules/soe_paragraph_image_cta/templates/image-cta-paragraph.html.twig

```diff
@@ -0,0 +1,26 @@
+
+{#
+/**
+ * @file
+ * Image CTA Paragraph
+ *
+ * Template for the Image CTA paragraph type.
+ *
+ * Available variables:
+ *
+ * {{ image }}
+ * {{ link }}
+ *
+ */
+#}
+
+{%- set attributes = attributes.addClass(['su-image-cta-paragraph', modifier_class]) -%}
+
+<div{{ attributes }}>
+    {# kint() #}

```

Comment:
Please remove debugging code prior to review.

Reply:
fixed.